index.css

```diff
@@ -16,415 +16,635 @@ main {
+}
 
-  .fa-bars {
-    color: blue;
-    padding-right: 14%;
-  }
+.fa-bars {
+  color: blue;
+  padding-right: 14%;
+}
 
-  .fa-bars:hover {
-    opacity: 0.5;
-  }
+.fa-bars:hover {
+  opacity: 0.5;
+}
 
-  .catch-text-sec {
-    height: 604px;
-    background-image: url(./assets//headerTitle.svg);
-    background-repeat: no-repeat;
-    background-size: cover;
-    background-color: #fff;
-    border-radius: 0 0 0 100px;
-    margin-bottom: 80px;
-  }
+.nav ul {
+  display: none;
+}
 
-  /* CATCH TEXT SECTION */
-  #catch-text-heading {
-    font-family: 'Poppins', sans-serif;
-    font-style: normal;
-    font-weight: 700;
-    font-size: 40px;
-    line-height: 52px;
-    display: flex;
-    align-items: center;
-    padding-top: 21%;
-    width: 327px;
-    height: 150px;
-    color: #172b4d;
-  }
+.catch-text-sec {
+  height: 604px;
+  background-image: url(./assets//headerTitle.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-color: #fff;
+  border-radius: 0 0 0 100px;
+  margin-bottom: 80px;
+}
 
-  .catch-text-sec > * {
-    margin-top: 0 7%;
-    padding-left: 9%;
-  }
+/* CATCH TEXT SECTION */
+#catch-text-heading {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  padding-top: 21%;
+  width: 327px;
+  height: 150px;
+  color: #172b4d;
+}
 
-  .about {
-    width: 327px;
-    height: 146px;
-    font-family: 'Poppins', sans-serif;
-    font-weight: 400;
-    font-size: 16px;
-    line-height: 24px;
-    color: #344563;
-    font-style: normal;
-  }
+.catch-text-sec > * {
+  margin-top: 0 7%;
+  padding-left: 9%;
+}
 
-  /* .social-icon */
-  .socials-header {
-    font-family: 'Poppins', sans-serif;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 16px;
-    line-height: 24px;
-    color: #7f8cff;
+.about {
+  width: 327px;
+  height: 146px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  font-style: normal;
+}
+
+/* .social-icon */
+.socials-header {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.socials-list {
+  list-style: none;
+  margin-left: -60px;
+  display: flex;
+}
+
+.social-icon {
+  float: left;
+  margin-left: 20px;
+  color: #505f79;
+}
+
+.social-icon:hover {
+  color: #396df2;
+}
+
+/* PROJECT SECTION */
+.works-sec {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.work-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 327px;
+  height: 552px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px;
+  align-items: center;
+  background: #fff;
+  margin: 0 10% 88px 10%;
+}
+
+.lng-wrk-card {
+  height: 596px;
+}
+
+.work-image-cont {
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+}
+
```

Comment:
- [ ] As suggested by the previous code reviewers, I think it is better if you adjust the style of your cards and the images to match the Figma design. :relaxed: 
  - As you can see from your design the height of all your card images is longer than the card itself. :smile:
  
### Your design
![Screenshot from 2022-12-10 23-24-15](https://user-images.githubusercontent.com/92303891/206879284-06b0760e-fc12-4cfd-a1ba-ab7beb6b5766.png)

### Figma design
![image](https://user-images.githubusercontent.com/92303891/206879290-0954befe-78ab-4a31-8e69-52665f01c2af.png)